packages/forklift-console-plugin/src/mock-console-extension/index.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+import { type SetupWorker, setupBrowserWorker } from '@kubev2v/mocks/src/console-plugin';
```

Comment:
the worker should be on the plugin side, the mock package only provide a method that return data when given the path, method... (request params ...) as props

Reply:
Moved the mock service worker setup to the console-plugin.